compiler/visq/visqlib/QErrorComputer.py

```diff
@@ -211,10 +211,15 @@ def run(self):
 class SRMSEComputer(QErrorComputer):
     def __init__(self, fp32_dir, fq_dir):
         super().__init__(fp32_dir, fq_dir)
-        self.scale_file = Path(fq_dir) / 'scales.txt'
+        if fq_dir != None:
```

Comment:
Is there a case when `fq_dir` is None?

Reply:
> Is there a case when `fq_dir` is None?

Yes.
See https://github.com/Samsung/ONE/blob/16dcb2f481ca650c6a4a49817d751ad84f3e039f/compiler/visq/visq#L364.

Reply:
Ah.. right. I think we have to change the constructor signature. Thanks!